.vscode/settings.json

```diff
@@ -23,9 +23,9 @@
     "[javascript][typescript][javascriptreact][typescriptreact]": {
         "editor.formatOnSave": true,
         "editor.codeActionsOnSave": {
-            "source.organizeImports": "always",
-            "source.formatDocument": "always",
-            "source.fixAll.eslint": "always"
+            "source.organizeImports": "explicit",
+            "source.formatDocument": "explicit",
+            "source.fixAll.eslint": "explicit"
```

Comment:
@kaf-lamed-beyt Why did you change this?

Reply:
I made a mistake while setting up vscode on my PC. so I went and copied this vscode settings.json file as is from my branch.